src/one_dragon_qt/widgets/multi_selection_combo_box.py

```
@@ -0,0 +1,548 @@
+        3. 根据计算的宽度设置父组件的 minimumWidth
+        4. 将标签添加到 FlowLayout，此时 FlowLayout 会按照正确的宽度进行布局
+
+        这样可以避免标签提前换行的问题。
+        """
+        if not self._tags:
+            return
+
+        parent = self.parent()
+        # 仅在动态宽度模式下调整宽度（非 fixed_width 模式）
+        if parent and hasattr(parent, '_fixed_width') and parent._fixed_width is None:
+            # 步骤1：计算所有标签需要的总宽度
```

Ruff (0.14.10)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)